service/src/main/java/tds/config/web/resources/TimeLimitsResource.java

```diff
@@ -0,0 +1,19 @@
+package tds.config.web.resources;
+
+import org.springframework.hateoas.ResourceSupport;
+import tds.config.TimeLimits;
+
+/**
+ * A HATEOAS representation of the {@link tds.config.TimeLimits} for return to the consumer.
+ */
+public class TimeLimitsResource extends ResourceSupport {
```

Comment:
Actually, I'm not following why we need a separate object that simply wraps in TimeLimits object.